accessibility-scan.yml

```diff
@@ -0,0 +1,25 @@
+jobs:
+  - name: accessibility-scan
+    plan:
+    - task: run-pa11y
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            repository: node
+        run:
+          path: sh
+          args:
+          - -exc
+          - |
+            npm install -g pa11y-crawl
```

Comment:
oo.. probably a good idea. We could also find a specific set of versions that are compatible with one another and install those specifically. I'm not sure if that is the _right_ solution, but an option for sure